File: 151004/Bordilovsky/DC_JPA/Services/Mappers/UserMapper.cs
using AutoMapper;
using DC_REST.DTOs.Request;
using DC_REST.DTOs.Response;
using DC_REST.Entities;

namespace DC_REST.Services.Mappers
{
	public class UserMapper : Profile
	{
		public UserMapper() 
		{
			CreateMap<User, UserResponseTo>();
			CreateMap<UserRequestTo, User>();
		}
	}
}

File: 151001/Yankova/RV2/src/main/java/com/bsuir/nastassiayankova/mappers/UserMapper.java
package com.bsuir.nastassiayankova.mappers;

import com.bsuir.nastassiayankova.beans.entity.User;
import com.bsuir.nastassiayankova.beans.response.UserResponseTo;
import org.mapstruct.Mapper;
import com.bsuir.nastassiayankova.beans.request.UserRequestTo;

@Mapper(componentModel = "spring", uses = NewsListMapper.class)
public interface UserMapper {
    User toUser(UserRequestTo userRequestTo);

    UserResponseTo toUserResponseTo(User user);
}
